editor/plugins/gdextension_export_plugin.h

```diff
@@ -77,24 +77,37 @@ void GDExtensionExportPlugin::_export_file(const String &p_path, const String &p
 
 	HashSet<String> archs;
 	HashSet<String> features_wo_arch;
+	Vector<String> features_vector;
 	for (const String &tag : p_features) {
 		if (all_archs.has(tag)) {
 			archs.insert(tag);
 		} else {
 			features_wo_arch.insert(tag);
 		}
+		features_vector.append(tag);
 	}
 
 	if (archs.is_empty()) {
 		archs.insert("unknown_arch"); // Not archs specified, still try to match.
 	}
 
 	HashSet<String> libs_added;
+	struct FoundLibInfo {
+		int count = 0;
+		Vector<String> libs;
+	};
+	HashMap<String, FoundLibInfo> libs_found;
```

Comment:
Perhaps a comment would be good that libs_found only exists for warning purposes.